server/url_params.go

```diff
@@ -63,6 +67,77 @@ func ExtractParametersFromUrl(url *url.URL) (params URLParameters, err error) {
 		params.pref.Builders = targetBuildersQuery
 	}
 
+	refundAddressQuery, ok := url.Query()["refund"]
+	if ok {
```

Comment:
why not invert this and reduce the amount of nested code? i.e.

```
if !ok {
    return params, nil 
}

# rest of logic here 
```

Reply:
I think it is better like this because if you look above - its how other queries are done. This `!ok return` can work only for the last query. So if we add another query here we would still need to change to 'if ok` like here.